<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuranReader</class>
 <widget class="QWidget" name="QuranReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>Chapter:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboChapter"/>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>Verses:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnVerseFrom">
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>286</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnVerseTo">
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>286</number>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="text">
     <string>Verse:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnVerse"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
